docs/cmdlet-example/README.md

```diff
@@ -0,0 +1,57 @@
+# Building a C# Cmdlet
+
+This demonstrates how to build your own C# cmdlet for PowerShell Core with Visual Studio.
+
+We will use free Visual Studio Community 2017 that can be downloaded from [https://www.visualstudio.com/downloads](https://www.visualstudio.com/downloads)
+
+1. When installing Visual Studio 2017 select `.NET Core cross-platform development` under `Other Toolsets`
+![1](https://user-images.githubusercontent.com/11860095/31471799-7ca192f4-ae9f-11e7-9731-03fb24ba949c.png)
+
+2. Create new C# project `SendGreeting` of type `Class Library (.NET Core)`
```

Comment:
I believe the markdown rules are to use `1.` for each item in the numbered list and the renderer takes care of incrementing

Reply:
I've checked - both syntaxes can be used.
Tried to use '1.' for all items, but got across formatting issues and number list getting broken after code section. Not worth the time spent on changing this.